platform-python.spec

```diff
@@ -2,46 +2,12 @@
 # Conditionals and other variables controlling the build
 # ======================================================
 
-# NOTES ON BOOTSTRAPING PYTHON 3.6:
-#
-# Due to a dependency cycle between Python, gdb, rpm, pip, setuptools, wheel,
-# and other packages, in order to rebase Python 3 one has to build in the
-# following order:
-#
-# 1. At the same time:
-#     - gdb without python support (add %%global _without_python 1 on top of gdb's SPEC file)
-#     - python-rpm-generators with bootstrapping_python set to 1
-#       (this can be done also during step 2., but should be done before 3.)
-# 2. python3 with with_rewheel set to 0
-# 3. At the same time:
-#     - gdb with python support (remove %%global _without_python 1 on top of gdb's SPEC file)
-#     - python-rpm-generators with bootstrapping_python set to 0
-#       (this can be done at any later step without negative effects)
-# 4. rpm
-# 5. python-setuptools with bootstrap set to 1
-# 6. python-pip with build_wheel set to 0
-# 7. python-wheel with %%bcond_without bootstrap
-# 8. python-setuptools with bootstrap set to 0 and also with_check set to 0
-# 9. python-pip with build_wheel set to 1
-# 10. pyparsing
-# 11. python3 with with_rewheel set to 1
-#
-# Then the most important packages have to be built, starting from their
-# various leaf dependencies recursively. After these have been built, a
-# targeted rebuild should be requested for the rest.
-#
-# Currently these packages are recommended to have been built before a targeted
-# rebuild after a python abi change:
-#   python-sphinx, pytest, python-requests, cloud-init, dnf, anaconda, abrt
-
-%global with_rewheel 1
-
```

Comment:
It might be useful to link to the Platform Python Stack change here on top, for quick reference.